examples/annotation_import/pdf.ipynb

```diff
@@ -173,7 +173,10 @@
         "  tools=[ \n",
```

Comment:
<p>Tool Type should RELATIONSHIP here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
thanks https://github.com/Labelbox/labelbox-python/pull/921/commits/69373ed1423c9d32c7b2de9332fa4fcd91260d64